### ISSUE TITLE
[Feature Request] Show full task text as tooltip if the task name is shortened

### ISSUE THREAD

**Issue by darthmachina** [roovo/obsidian-card-board#23]
I like the idea of shortening a longer task description, but it would be nice to be able to see what the full text is. My proposal is to add a tooltip that will show the full task text.

I would probably add a delay between the user stopping the mouse on the text and showing the tooltip as it might be annoying if the tooltip kept showing up when using the mouse normally.

**Comment by darthmachina** [comment]
I did some looking into this one since I figured it'd be fairly straightforward and wouldn't get impacted much by the refactor. Turns out, both yes and no. 

It's dead simple to get the tooltip in place, without even any custom CSS. Can use an `aria-label` attribute which is what Obsidian uses for its tooltips and it works just like in the rest of Obsidian. However, there is no delay, just like in the rest of Obsidian. Can also use the `title` attribute which will show an OS specific tooltip. There is a delay but the tooltip is not styled.

However, I just can't get pure CSS that will only show the tooltip if the ellipsis is shown, but only then. It's an all-or-none thing right now. This is what it looks like with `aria-label`:
![image](https://user-images.githubusercontent.com/3865380/140006177-eca86e40-5180-4754-92fc-b0664bfaa663.png)

To be honest, with Drag and Drop potentially coming at some point, I'm not really sure this is worth it since both will compete for the mouse cursor on a card. With indented list items being shown as notes on the task and knowing that you get a certain amount of space I would probably just craft my task names to be shorter and use indented items for more information.
